### PR TITLE
helm: set reana-server API limiter configs

### PR DESCRIPTION
**Pull request by VMois**
closes reanahub/reana-server#404

Merge after https://github.com/reanahub/reana-server/pull/412 (to be able to test)

### CODE REVIEW

File: helm/reana/values.yaml [hunk 1]
```diff
@@ -67,6 +67,8 @@ components:
       REANA_SCHEDULER_REQUEUE_SLEEP: 15
       REANA_USER_EMAIL_CONFIRMATION: true
       REANA_WORKFLOW_SCHEDULING_POLICY: "fifo"
+      REANA_RATELIMIT_GUEST_USER: "20 per second"
+      REANA_RATELIMIT_AUTHENTICATED_USER: "20 per second"
```
Comment (mvidalgarcia) [inline helm/reana/values.yaml:71]: If these are [the default values in reana-server](https://github.com/reanahub/reana-server/pull/412/files#diff-493aa29a266bfb1026442110e526ad9030f21f4d0e4ea10bdf1ae4c5dafe491dR159-R162), you don't even need to set them here.

File: CHANGES.rst [hunk 2]
```diff
@@ -12,6 +12,7 @@ Version 0.8.0 (UNRELEASED)
     - Adds a default ``kubernetes_memory_limit`` value (4Gi).
     - Adds configuration environment variable to set workflow scheduling policy (``REANA_WORKFLOW_SCHEDULING_POLICY``).
     - Changes Helm templates to use PostgreSQL 12.8 version.
+    - Adds configuration environment variable to set API limiter configs (``REANA_RATELIMIT_GUEST_USER``, ``REANA_RATELIMIT_AUTHENTICATED_USER``).
```
Comment (mvidalgarcia) [inline CHANGES.rst:15]: We usually group changelog items per verb, i.e. All the "Adds..." together, all the "Changes..." and so on.

Comment (VMois) [reply CHANGES.rst:14]: Done

File: helm/reana/values.yaml (hunk 1 above)
Comment (VMois) [reply helm/reana/values.yaml]: Done

File: CHANGES.rst (hunk 2 above)
Comment (mvidalgarcia) [reply CHANGES.rst:14]: Not sure if the word "configs" here is the more appropriate one. We can be more descriptive as this is targeted at administrators.

Comment (VMois) [reply CHANGES.rst:14]: I agree. Not sure about what can be a better name. Some options:

- `...API rate limit values`
- `...API rate limiting characteristics` (taken from `flask-limiter` docs page)

P.S need to change `variable` to `variables` :)

Comment (tiborsimko) [reply CHANGES.rst:14]: "Adds configuration environment variables to set REST API rate limit values (VAR1, VAR2)" looks good...

Comment (VMois) [reply CHANGES.rst]: Done

**Review by mvidalgarcia — APPROVED** [review]

**Pull request merged**